Packs/ExportIndicators/ReleaseNotes/1_0_11.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Export Indicators Service
+- Fixed an issue where indicators where pulled using the *n* argument instead of the *List Size* parameter before sorting them, causing unexpected results.
```

Comment:
```suggestion
- Fixed an issue where indicators were pulled using the *n* URL line argument instead of the *List Size* integration parameter before sorting them, causing unexpected results.
```